parm/config/gfs/yaml/defaults.yaml

```diff
@@ -13,13 +15,13 @@ landanl:
   IO_LAYOUT_Y: 1
 
 ocnanal:
-  SOCA_INPUT_FIX_DIR: '/scratch2/NCEPDEV/ocean/Guillaume.Vernieres/data/static/72x35x25'
-  CASE_ANL: 'C48'
-  SOCA_OBS_LIST: ''
-  COMIN_OBS: '/scratch2/NCEPDEV/marineda/r2d2'
-  SABER_BLOCKS_YAML: ''
-  SOCA_NINNER: 50
-  R2D2_OBS_SRC: 'gdas_marine'
-  R2D2_OBS_DUMP: 's2s_v1'
+  SOCA_INPUT_FIX_DIR: "/scratch2/NCEPDEV/ocean/Guillaume.Vernieres/data/static/72x35x25/soca"  # TODO: These need to go to glopara fix space. @guillaumevernieres will open an issue
+  CASE_ANL: "C48"
+  COMIN_OBS: "/scratch2/NCEPDEV/marineda/r2d2-v2-v3"  # TODO: make platform agnostic
```

Comment:
Wary about merging this into develop until these are on both RDHPCS platforms

Reply:
it doesn't matter.  There is no soca test. Nothing will break

Reply:
See https://github.com/NOAA-EMC/GDASApp/issues/678